test/test-util.ts

```diff
@@ -16,19 +16,121 @@
 import test from 'ava';
 import * as path from 'path';
 
-import {getTSConfig} from '../src/util';
+import {ConfigFile, getTSConfig} from '../src/util';
+
+
+
+/**
+ * Creates a fake promisified readFile function from a map
+ * @param myMap contains a filepath as the key and a ConfigFile object as the
+ * value. The fake promisified readFile function has the same interface of
+ * fs.readFile and takes in a
+ * @param configPath and returns a corresponding ConfigFile object or throws an
```

Comment:
This `@param` annotation is a little misleading, since it's a parameter of the returned function, not this one. I think it's acceptable to just say that the return value has the same interface as a promisifed fs.readFile function.